packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/sourceDescription/TypeVisitor.ts

```diff
@@ -0,0 +1,17 @@
+import stampit from 'stampit';
+import { StringElement, BREAK, cloneDeep } from '@swagger-api/apidom-core';
+
+import FallbackVisitor from '../../FallbackVisitor';
+
+const TypeVisitor = stampit(FallbackVisitor, {
+  methods: {
+    StringElement(stringElement: StringElement) {
+      this.element = cloneDeep(stringElement);
+      this.element.classes.push('sourceDescription-type');
```

Comment:
If you'll use classes here, the convention is to use `dash-case`: 
```suggestion
      this.element.classes.push('source-description-type');
```